README.md

```diff
@@ -3,34 +3,85 @@
 
 # Maliput
 
-Road network runtime interface.
+## Description
 
-## Build
+A C++ runtime API describing a Road Network model for use in agent and traffic simulations. It guarantees a continuous description of the road geometry and supports dynamic environments with varying rules states.
 
-1. Setup a development workspace as described [here](https://maliput.readthedocs.io/en/latest/developer_setup.html).
+For a full overview of Maliput capabilities please visit https://maliput.readthedocs.io/en/latest/maliput_overview.html.
 
-2. Bring up your development workspace:
+## API Documentation
 
-```sh
-cd path/to/my/workspace
-source ./bringup
+Refer to [Maliput's Online API Documentation](https://maliput.readthedocs.io/en/latest/html/deps/maliput/html/annotated.html).
+
+## Examples
+
+[Getting Started](https://maliput.readthedocs.io/en/latest/getting_started.html) page is a good place for starting to see the maliput's capabilities.
+
+There are a couple of packages where the maliput's api is excercised.
+ - [maliput_integration](https://github.com/maliput/maliput_integration): Concentrates applications created for maliput. See [maliput_integration's tutorials](https://maliput.readthedocs.io/en/latest/html/deps/maliput_integration/html/integration_tutorials.html).
+ - [delphyne_demos](https://github.com/maliput/delphyne_demos): Contains demos based on delphyne, which is an agent simulation framework that uses `maliput` as road network model. See [delphyne_demos](https://github.com/maliput/delphyne_demos).
+
+
+## Installation
+
+### Supported platforms
+
+Ubuntu Focal Fossa 20.04 LTS.
+
+### Binary Installation on Ubuntu
+
+See [Installation Docs](https://maliput.readthedocs.io/en/latest/installation.html#binary-installation-on-ubuntu).
+
+### Source Installation on Ubuntu
+
+#### Prerequisites
+
+```
+sudo apt install python3-rosdep python3-colcon-common-extensions
 ```
 
-3. Build maliput packages and their dependencies:
+#### Build
+
+1. Create colcon workspace if you don't have one yet.
+    ```sh
+    mkdir colcon_ws/src -p
+    ```
+
+2. Clone this repository in the `src` folder
+    ```sh
+    cd colcon_ws/src
+    git clone https://github.com/maliput/maliput.git
+    ```
+
+3. Install package dependencies via `rosdep`
+    ```
+    export ROS_DISTRO=foxy
+    ```
+    ```sh
+    rosdep update
+    rosdep install -i -y --rosdistro $ROS_DISTRO --from-paths src
+    ```
+
+4. Build the package
+    ```sh
+    colcon build --packages-up-to maliput
+    ```
+
+    **Note**: To build documentation a `-BUILD_DOCS` cmake flag is required:
+    ```sh
+    colcon build --packages-select maliput --cmake-args " -DBUILD_DOCS=On"
+    ```
+
+For further info refer to [Source Installation on Ubuntu](https://maliput.readthedocs.io/en/latest/installation.html#source-installation-on-ubuntu)
```

Comment:
Thanks for your weigh in, I see your point, it doesn't add up to the source installation for this package.

However, I added that link just for providing enough links to general documentation, which is under development, and probably in the future that source installation section in https://maliput.readthedocs.io/en/latest/installation.html#source-installation-on-ubuntu may contain more valuable information.

Furthermore, I wouldn't like to delete this section and point directly to the developer setup because, there, we compile all the repos from source. While here we are relying on rosdep for installing the maliput's dependencies (It will be more evident when I update the readme of a downstream package like maliput_malidrive)). So if there is a user that want to compile from source just one package for some reason, this will be quite more simple than diving into the developer setup page.